src/vfs.js

```diff
@@ -79,5 +79,18 @@ fs.writeFileSync(
   "third_party/vue.runtime.esm-nobuble-2.4.4.js",
   require("raw-loader!../third_party/vue.runtime.esm-nobuble-2.4.4.js")
 );
+fs.mkdirpSync("/src/fixtures/webpack");
```

Comment:
Webpack requires absolute file paths.

Reply:
How does that work when you run it via `node`? I.e.

```bash
$ node src/cli
```